provider/cluster_rosa_classic_resource.go

```diff
@@ -1657,7 +1657,7 @@ func populateRosaClassicClusterState(ctx context.Context, object *cmv1.Cluster,
 	}
 
 	trustBundle, ok := object.GetAdditionalTrustBundle()
-	if ok {
+	if ok && common.IsStringAttributeEmpty(state.Proxy.AdditionalTrustBundle) {
```

Comment:
Please also open a PR for 1.2-prerelease branch (we would like to publish a fix)

Reply:
Cant we cherry-pick after the merge?

Reply:
you can type right now ``<slash>cherry-pick v1.2.0-prerelease.1-branch`` and it will get applied after the merge

Reply:
Also, please add a commit with the changelog changes...